tests/benchmarks/README.md

```diff
@@ -1,13 +1,86 @@
 # Automated Benchmarking
 [Benchmark repo](https://github.com/rlworkgroup/benchmarks)
 
-We have an automated system that will run any benchmark file named "test\_benchmark_*.py" in this folder /tests/benchmarks/.
+We have an automated system that will run any benchmark file named "test_benchmark_*.py" in this folder /tests/benchmarks/.
 
 The results are displayed [here](https://rlworkgroup.github.io/benchmarks/)
 
 # Directions
     Follow these directions to output your benchmark results as a json compatible with the display.
-    1. import tests.helpers as Rh
+    1. from tests.helpers import create_json, write_file, plot
     2. Create a result_json = {}
-    3. for each task: result_json [task] = Rh.create_json()
-    4. Rh.write_file(result_json, 'Algo name')
+    3. for each task: result_json[task] = create_json()
+    4. write_file(result_json, 'Algo name')
+
+# Example
+    '''python
+     def test_benchmark_ddpg(self):
+        #
+        #
+        #
+        result_json = {}
+        for task in mujoco1m['tasks']:
+            #
```

Comment:
What are these commented out blank lines?

Reply:
I thought it might be easier to read, instead of including the entire test_benchmark_ddpg
 function